theme.json

```diff
@@ -612,23 +612,8 @@
 			},
 			"core/pullquote": {
 				"border": {
-					"bottom": {
-						"style": "none",
-						"width": "0px"
-					},
-					"left": {
-						"style": "none",
-						"width": "0px"
-					},
 					"radius": "var(--wp--preset--spacing--20)",
-					"right": {
-						"style": "none",
-						"width": "0px"
-					},
-					"top": {
-						"style": "none",
-						"width": "0px"
-					}
+					"style": "solid"
```

Comment:
Please remove the `"style": "solid"` too, as it creates a border that is not in the design:

<img width="648" alt="CleanShot 2023-10-04 at 13 07 38@2x" src="https://github.com/WordPress/twentytwentyfour/assets/7585600/c7e15654-a503-4d00-84c2-af0e61320af2">

Reply:
Hi @luminuu 

I have updated my PR and removed the above-mentioned line.

Please review it.

Thanks,